src/Network/Email/AbstractTransport.php

```diff
@@ -1,3 +1,16 @@
 <?php
-// @deprecated Backward compatibility with 2.x, 3.0.x
-class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+// @codingStandardsIgnoreFile
+namespace Cake\Network\Email;
+
+class_alias(\Cake\Mailer\AbstractTransport::class, AbstractTransport::class);
+
+if (class_exists(AbstractTransport::class)) {
+    return;
+}
```

Comment:
@chinpei215 actually, it's been reported as a bug in phpstorm for not support class_alias. That is the real problem. If that worked properly then none of this would be needed. They've not closed the issue for 2 years. So I don't expect a solution anytime soon.

After after class_alias is a hack.

If we do not do this it breaks all my intellisense. All the network classes have been moved, and the IDE reports the old usages as undefined.

Keep in mind that these empty class files will only be loaded for projects that have not migrated their code to the new classes. If there is any concern about performance they should migrate, and they have to anyway because these are deprecated.